### PR TITLE
DSValidator Refactored

### PR DESCRIPTION
**Pull request by ssvlab**

### CODE REVIEW

File: toolbox-dsvalidator/toolbox/reproduceMinimumPhase.m [hunk 1]
```diff
@@ -54,6 +54,6 @@ function reproduceMinimumPhase(path, varargin)
 
 property = 'm';
 
-dsv_validation(path, property, ovmode, rmode, filename);
```
Comment (lucasccordeiro) [inline toolbox-dsvalidator/toolbox/reproduceMinimumPhase.m:57]: Why did you change the interface?

File: toolbox-dsvalidator/toolbox/reproduceLimitCycle.m [hunk 2]
```diff
@@ -54,6 +54,6 @@ function reproduceLimitCycle(path, varargin)
 
 property = 'lc';
 
-dsv_validation(path, property, ovmode, rmode, filename);
```
Comment (lucasccordeiro) [inline toolbox-dsvalidator/toolbox/reproduceLimitCycle.m:57]: Why did you change the interface?

File: toolbox-dsvalidator/toolbox/reproduceStateSpaceStability.m [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+var = varargin;
+
+if nvar >= 2
+if length(var{1}) > 0
+ ovmode = var{1};
+end
+end
+
+if nvar >= 3
+if length(var{2}) > 0
```
Comment (lucasccordeiro) [inline toolbox-dsvalidator/toolbox/reproduceStateSpaceStability.m:44]: Fix indentation?

File: toolbox-dsvalidator/toolbox/reproduceStateSpaceObservability.m [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+ovmode = '';
+rmode = '';
+filename = '';
+
+nvar = nargin;
+var = varargin;
+
+if nvar >= 2
+if length(var{1}) > 0
```
Comment (lucasccordeiro) [inline toolbox-dsvalidator/toolbox/reproduceStateSpaceObservability.m:38]: Fix indentation?

File: toolbox-dsvalidator/toolbox/reproduceStateSpaceControllability.m [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+end
+
+if nvar >= 3
+if length(var{2}) > 0
+ rmode = var{2};
+end
+end
+
+if nvar >= 4
+if length(var{3}) > 0
```
Comment (lucasccordeiro) [inline toolbox-dsvalidator/toolbox/reproduceStateSpaceControllability.m:50]: fix indentation?

**Comment by lennonchaves** [comment]
I'm working right now in some changes in the DSValidator scripts (inclusion of stability in closed-loop and state-space reproducibility), and refactoring code (updating comments, functions, renaming variables...). In the next version, I'll fix the indentation and comments. Many Thanks.
